locales/en/metrics.json

```diff
@@ -53,5 +53,12 @@
   "total_bytes": "Bytes incoming and outgoing",
   "total_bytes_popover_header": "Bytes incoming and outgoing",
   "used_disk_space": "Used disk space",
-  "used_disk_space_help_text": "Used disk space is the amount of disk space used by the Kafka broker in the instance. This metric enables you to assess available disk space relative to the limit. To reduce used disk space, you can adjust topic retention time or other topic properties as needed."
+  "used_disk_space_help_text": "Used disk space is the amount of disk space used by the Kafka broker in the instance. This metric enables you to assess available disk space relative to the limit. To reduce used disk space, you can adjust topic retention time or other topic properties as needed.",
+  "partition_limit": "Limit 1000 partitions.",
+  "partition_limit_approaching_description_1": "This Kafka Instance is approaching the partition limit. If the Kafka Instance exceeds 1000 partitions, it might experience degraded performance.",
```

Comment:
* Please change "Instance" to "instance" with a lower-case "i"
* Please replace `1000` with a value pulled from the API for the actual instance limit